carma_ros2_utils/package.xml

```diff
@@ -13,6 +13,7 @@
   <build_depend>carma_cmake_common</build_depend>
```

Comment:
Does the build still depend on carma_cmake_common?

Reply:
Yes. That's where the carma_package and ros check macros come from